docs/src/main/paradox/typed/guide/modules.md

```diff
@@ -233,7 +233,7 @@ Distributed Data is intended to solve the following challenges:
   symbol1=PekkoVersion
   value1="$pekko.version$"
   group=org.apache.pekko
-  artifact=akka-stream-typed_$scala.binary.version$
```

Comment:
There is no such thing as akka-stream-typed, this was incorrect even before all of the pekko related changes.